homeassistant/components/keenetic_ndms2/config_flow.py

```diff
@@ -40,17 +43,20 @@ class KeeneticFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry):
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> KeeneticOptionsFlowHandler:
         """Get the options flow for this handler."""
         return KeeneticOptionsFlowHandler(config_entry)
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: dict[str, any] | None = None
+    ) -> dict[str, any]:
```

Comment:
```suggestion
    ) -> dict[str, Any]:
```

Reply:
Now. THis is confusing.... as part of the previous review steps you proposed using 'any', now the imported 'Any'. What's the point? Can you explain?